tasks/install.yml

```
@@ -122,17 +126,15 @@
     nonsensitive_stdout.skipped is not defined
     or not nonsensitive_stdout.skipped | bool
 
+- name: Install | Remove state file
```

nitpick, suggested step rename:

```suggestion
- name: Install | Revert State Upon Error
```